### ISSUE TITLE
Colors get stripped

### ISSUE THREAD

**Issue by ericelliott** [mikeal/watch#69]
I have the following scripts:

```
  "scripts": {
    "lint": "jshint .",
    "test": "tape test/index.js | tap-bail | tap-difflet",
    "c1:section5:example-1": "tape examples/software-testing/section5/example-1.js | tap-bail | tap-difflet",
    "c1:section5:example-2": "tape examples/software-testing/section5/example-1.js | tap-bail | tap-difflet",
    "dev": "watch --wait=5 'npm run lint && npm run test' ."
  }
```
When I run these scripts directly, they have color output (via tap-difflet). When I do `npm run dev` and use `watch` to launch the tasks, the colors are stripped.

**Comment by finnp** [comment]
I guess this is because tap-difflet does strip colors when not run as a tty. I am fixing this right now.

**Comment by finnp** [comment]
This should be fixed in 0.14.0. Can you confirm this?

**Comment by ericelliott** [comment]
Seems to be working now, thanks!
